examples/with-relay-modern/README.md

```diff
@@ -1,49 +0,0 @@
-# Relay Modern Example
```

Comment:
Let's not delete this file and rather mention that this is now moved.

Reply:
I find this counter productive. It's will only add more confusion to new comers now that "relay modern" is just "relay".

Reply:
The reason we don't want to remove old examples is that there might exist a lot of links out there that still point to that location. We want the developer to know that the example has moved.

Reply:
So remove all the example code, but leave a single README with a link to the new example?

Reply:
Correct :+1: